engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/VolumeOrchestrator.java

```diff
@@ -141,7 +141,6 @@
 import com.cloud.vm.dao.UserVmDao;
 
 public class VolumeOrchestrator extends ManagerBase implements VolumeOrchestrationService, Configurable {
-
```

Comment:
makes sense but will lead possibly to unecessary conflicts during rebases, which are inevitable for this PR